PurchasesTests/SwiftSources/LocalReceiptParsing/DataConverters/UInt8+ExtensionsTests.swift

```diff
@@ -0,0 +1,38 @@
+import XCTest
+import Nimble
+
+@testable import Purchases
+
+class UInt8ExtensionsTests: XCTestCase {
+    
+    func testBitAtIndexGetsCorrectValue() {
+        expect(UInt8(0b10000000).bitAtIndex(0)) == 1
+        expect(UInt8(0b10000000).bitAtIndex(1)) == 0
+        expect(UInt8(0b00100000).bitAtIndex(2)) == 1
+        expect(UInt8(0b11111111).bitAtIndex(3)) == 1
```

Comment:
Since all of them are 1, the test is not really proving that it's getting the right one right? If I understand correctly, this should be:
```
        expect(UInt8(0b00010000).bitAtIndex(3)) == 1
```

I might be getting this all wrong lol. Also, it's missing the case where you check for the 8th bit, you're only checking for 7 of the bits.

I don't think it's a big deal though